modules/gapi/include/opencv2/gapi/imgproc.hpp

```diff
@@ -847,6 +876,36 @@ Output image must be 8-bit unsigned 2-channel image @ref CV_8UC2.
 */
 GAPI_EXPORTS GMat RGB2YUV422(const GMat& src);
 
+/** @brief Converts an image from NV12 (YUV420p) color space to RGB.
+The function converts an input image from NV12 color space to RGB.
+The conventional ranges for Y, U, and V channel values are 0 to 255.
+
+Output image must be 8-bit unsigned planar 3-channel image @ref CV_8UC3.
```

Comment:
> planar 3-channel

`CV_8UC3` is interleaved 3-channel.

Reply:
I've upadted doxygen comments. I suggest to merge it as is to move forward.